src/dsp_tools/utils/connection_live.py

```diff
@@ -314,7 +318,7 @@ def _log_and_raise_timeouts(self, error: TimeoutError | ReadTimeout) -> None:
         msg = f"A '{error.__class__.__name__}' occurred during the connection to the DSP server."
         print(f"{datetime.now()}: {msg}")
         logger.exception(msg)
-        raise PermanentConnectionError(msg) from None
```

Comment:
Because the function `handle_non_ok_responses` also raises the `PermanentConnectionError` now the upload stopped with the new message of the resume upload even if for example a cardinality of a resource is wrong. This re-established the old way of dealing with those errors.